control-plane/pkg/core/config/broker_test.go

```diff
@@ -0,0 +1,236 @@
+package config
```

Comment:
More than calling this module `core/config`, how about naming it `dataplane/config`?

Reply:
Can we do this in another PR? It could be part of the naming improvement of #142. WDYT?

Reply:
well, you created that package here, do the change here :smile: